### PR TITLE
[feature] ft-setup-api 

### PR DESCRIPTION
**Pull request by danielogen**
# API setup and initial routes 

**_WHAT DOES THIS PR DO?_**
This PR sets up API using python flask micro-framework and created three endpoints 
**_WHAT TRELL STORY/TASK IS RELATED TO THIS PR?_**
N/A

**_WHAT IS THE LINK TO THE PR BRANCH_**
https://github.com/Kilega/Kilega-api/tree/ft-api-setup
**_HOW DO I TEST OUT THIS PR?_**
Git clone and checkout this branch locally. Create python `virtual env` and install packages using `pip install -r requirements/dev.txt`. Using any REST client of your choice, test the endpoints below;
- NB: you also need to setup the `DB` locally.
 
**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
- http://localhost:4001/
- http://localhost:4001/health
- http://localhost:4001/api/v1/songs?lang=acholi

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
N/A
**_ARE THERE ANY RELATED PRs?_**
N/A

### CODE REVIEW

**Comment by danielogen** [comment]
updated this PR

**Comment by danielogen** [comment]
Test comment

File: src/requirements/prod.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+attrs==20.3.0
+blinker==1.4
+click==7.1.2
+coverage==5.4
+Flask==1.1.2
+Flask-Cors==3.0.10
+Flask-Mail==0.9.1
```
Comment (danielogen) [inline src/requirements/prod.txt:7]: added email package

File: src/requirements/prod.txt [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+itsdangerous==1.1.0
+Jinja2==2.11.3
+MarkupSafe==1.1.1
+packaging==20.9
+pluggy==0.13.1
+py==1.10.0
+pymongo==3.11.3
+pyparsing==2.4.7
+pytest==6.2.2
+pytest-cov==2.11.1
```
Comment (danielogen) [inline src/requirements/prod.txt:19]: add code coverage package

**Review by OgenrwotAaron — APPROVED** [review]

**Pull request merged**